app/DTO/AlbumProtectionPolicy.php

```diff
@@ -2,50 +2,71 @@
 
 namespace App\DTO;
 
-class AlbumProtectionPolicy extends DTO
-{
-	public const IS_PUBLIC_ATTRIBUTE = 'is_public';
-	public const REQUIRES_LINK_ATTRIBUTE = 'requires_link';
-	public const IS_NSFW_ATTRIBUTE = 'is_nsfw';
-	public const IS_DOWNLOADABLE_ATTRIBUTE = 'is_downloadable';
-	public const IS_SHARE_BUTTON_VISIBLE_ATTRIBUTE = 'is_share_button_visible';
-	public const GRANTS_FULL_PHOTO_ATTRIBUTE = 'grants_full_photo';
-
-	public bool $isPublic;
-	public bool $requiresLink;
-	public bool $isNSFW;
-	public bool $isDownloadable;
-	public bool $isShareButtonVisible;
-	public bool $grantsFullPhoto;
+use App\Models\BaseAlbumImpl;
+use App\Models\Extensions\BaseAlbum;
+use App\SmartAlbums\BaseSmartAlbum;
 
+/**
+ * This represents the Album Protection policy.
+ *
+ * In other words it provides information on the security attributes of an album.
+ * It MUST not be used in the front-end to determine whether an action is permitted or not (e.g. share link available).
```

Comment:
```suggestion
 * It MUST NOT be used in the front-end to determine whether an action is permitted or not (e.g. share link available).
```

If we use capitalization to stress the modal verb we should do it right. The "NOT" part is the important one here.